citi-onboarding/Recorda-one-page
# Fix/navbar&max width

### *What I did:*
- fixed the navbar
- setted a max-width to all sections

### *How to check:*
- git checkout main
- git pull
- git checkout fix/navbar"&"max-width
- open the client on the terminal
- run npm start
- check the page
- review the code
